plugin-maven/src/test/java/com/diffplug/spotless/maven/generic/IndentTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.maven.generic;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+import com.diffplug.spotless.maven.MavenIntegrationTest;
+
+public class IndentTest extends MavenIntegrationTest {
+
+	@Test
+	public void fromContentToTabs() throws Exception {
+		writePomWithFormatSteps(
+				"<indent>",
+				"  <withTabs>true</withTabs>",
+				"</indent>");
+		runToTabTest();
+	}
+
+	@Test
+	public void fromContentToSpaces() throws Exception {
+		writePomWithFormatSteps(
+				"<indent>",
+				"  <withSpaces>true</withSpaces>",
+				"</indent>");
+		runToSpacesTest();
+	}
+
+	private void runToTabTest() throws Exception {
+		runTest("indent/IndentedWithSpace.test", "indent/IndentedWithTab.test");
+	}
+
+	private void runToSpacesTest() throws Exception {
+		runTest("indent/IndentedWithTab.test", "indent/IndentedWithSpace.test");
+	}
+
+	private void runTest(String source, String target) throws Exception {
+		setFile("src/main/java/test.java").toResource(source);
+		mavenRunner().withArguments("spotless:apply").runNoError();
+		String actual = read("src/main/java/test.java");
+		assertThat(actual).isEqualTo(getTestResource(target));
+	}
```

Comment:
`assertFile("src/main/java/test.java").sameAsResource(target)`

Reply:
I used the test structure of the existing plugin-maven tests, so I wasn't aware of this. Should I refactor the existing tests too?

Reply:
Ah, stuff slips through the cracks all the time, especially for a giant change like adding a whole new plugin.  It'd be great to have the existing tests refactored.  No obligation to do it though.